### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by JarrodPW**
@JakeEdm

Please review my code syntax, names and overall functionality and provide any feedback.

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    my_guitars.append(Guitar(name, year, cost))
+    print(f"{name} ({year}) : ${cost:,.2f} added")
+    name = input("Name: ")
+print("These are my guitars:")
+for i, guitar in enumerate(my_guitars, 1):
+    vintage_status = ""
```
Comment (JakeEdm) [inline prac_06/guitars.py:20]: can make the vintage_status into a single line of code
e.g. vintage_status = "(Vintage)" if guitar.is_vintage() else ""

**Review by JakeEdm — APPROVED** [review]
Looks good mate, one bit of advice is when commiting files to these branches dont include anything else but the .py files

Edit: Looks like you've deleted them already

File: prac_06/guitars.py (hunk 1 above)
Comment (JakeEdm) [reply prac_06/guitars.py:20]: Was lines 20-23, not what its showing above

Comment (JarrodPW) [reply prac_06/guitars.py:20]: Thank you for the feedback :)

File: prac_06/used_cars.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    # 2.
+    limo.add_fuel(20)
+    # 3.
+    print(limo.fuel)
+    # 4.
+    limo.drive(115)
+    # 5.
+    print(limo.odometer)
+    # 8.
+    print(limo.__str__())
```
Comment (lindsaymarkward) [inline prac_06/used_cars.py:29]: No need to use verbose syntax, just `print(limo)`

**Pull request merged**